prog/dftb+/lib_reks/reks.F90

```diff
@@ -0,0 +1,60 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2006 - 2020  DFTB+ developers group                                               !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+!> REKS and SI-SA-REKS formulation in DFTB as developed by Lee et al.
+!>
+!> The functionality of the module has some limitation:
+!> * Third order does not work.
+!> * Periodic system do not work yet appart from Gamma point.
+!> * Orbital potentials or spin-orbit or external E-field does not work yet.
+!> * Only for closed shell system.
+!> * Onsite corrections are not included in this version
+module dftbp_reks
+
+  use dftbp_rekscommon
+  use dftbp_reksen
+  use dftbp_reksfon
+  use dftbp_reksinterface
+  use dftbp_reksproperty
+  use dftbp_reksvar
+!  import dftbp_rekscommon
+!  import dftbp_reksen
+!  import dftbp_reksfon
+!  import dftbp_reksinterface
+!  import dftbp_reksproperty
+!  import dftbp_reksvar
```

Comment:
@aradi I created a high level module `dftbp_reks`, but it shows an error during compilation when I use `import` statement. Simply, I changed to `use` statement, and it shows no error and it works well. But, is this OK for this module? Actually, I am not familiar with `import` statement, so there can exist some mistake in the code. Please see this commit 2eee49d.